packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -77,6 +80,12 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
     state: 'selected',
   });
 
+  const getChildren = (id) =>
```

Comment:
Bit worried about performance here due to all the looping. Something to think about I guess.